data-service/src/main/java/uk/gov/gchq/palisade/service/data/DataApplication.java

```diff
@@ -50,36 +57,49 @@
 public class DataApplication {
     private static final Logger LOGGER = LoggerFactory.getLogger(DataApplication.class);
 
-    private final DataReader dataReader;
     private final AuditMessageService auditMessageService;
-
+    private final DataReader dataReader;
     private final StdSerialiserConfiguration serialiserConfiguration;
+
     private final RunnableGraph<Sink<TokenMessagePair, NotUsed>> runner;
```

Comment:
```suggestion
    private final RunnableGraph<Sink<TokenMessagePair, NotUsed>> runners;
```

Reply:
We only have one runnableGraph here, hence `runner` over the `Collection<..> runners` seen elsewhere